airbnb_project/airbnb_project/settings.py

```diff
@@ -80,12 +80,14 @@
 
 DATABASES = {
     'default': {
-        'NAME': 'airbnb_db',
         'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.environ.get('POSTGRES_DB', 'airbnb_db'),
+        'USER': os.environ.get('POSTGRES_USER', 'postgres'),
+        'PASSWORD': os.environ.get('POSTGRES_PASSWORD', '123456789'),
```

Comment:
it's always good to make it easily identifiable that something is a dummy value.

Reply:
In what way? Do I need to add a comment or docstring around here?

Reply:
you can pre-fix 'dummy' or 'do-not-use' as a keyword in the string itself. But let's throw an error to keep it more restrictive.